typescript/ws-protocol-examples/src/examples/mcap-record.ts

```diff
@@ -218,4 +243,11 @@ export default new Command("mcap-record")
   .description("connect to a WebSocket server and record an MCAP file")
   .argument("<address>", "WebSocket address, e.g. ws://localhost:8765")
   .option("-o, --output <file>", "path to write MCAP file")
+  .option("-n, --no-compression", "do not compress chunks")
+  .option(
+    "-q, --queue-limit <value>",
```

Comment:
Is "queue limit" a common phrase? I think I've heard "queue size" more frequently?